cmd/ctr/commands/images/import.go

```diff
@@ -34,45 +35,58 @@ var importCommand = cli.Command{
 	ArgsUsage: "[flags] <in>",
 	Description: `Import images from a tar stream.
 Implemented formats:
-- oci.v1     (default)
+- oci.v1
+- docker.v1.1
+- docker.v1.2
 
 
-For oci.v1 format, you need to specify --oci-name because an OCI archive contains image refs (tags)
-but does not contain the base image name.
+For OCI v1, you may need to specify --base-name because an OCI archive
+contains only partial image references (tags without the base image name).
+If no base image name is provided, a name will be generated as "import-%{date}".
```

Comment:
`date` -> `yyyy-MM-dd` ?

Reply:
Should we mention that the default platform will be assigned?

Reply:
I was trying to avoid platform related stuff in this PR, platform options should definitely be included as well as the option to not unpack.